CHANGELOG.rst

```diff
@@ -2,6 +2,353 @@
+
+An example is defined in :ref:`Prefill examples <examples_objects_prefill>`.
+
+**🖋️ Cosign flow improvements**
+
+We now provide a much more intuitive user experience to have someone cosign a form submission - users need
+to click less and in general we removed a lot of friction for this process.
+
+On top of that, the new configuration options for cosign allow you to tweak the content of emails and screens 
+when cosigning is enabled in a form - from inviting someone to cosign to the confirmation page they get.
+
+**💳 More powerful price calculations**
+
+We made it simpler and more intuitive for form designers to define dynamic price logic rules - these are now
+part of the regular logic rules. This also enables you to perform more complex calculations and interact with
+external systems to retrieve pricing information!
+
+**🛑 Limiting the amount of submissions**
+
+You can now specify a maximum number of submissions for a given form, useful for limited availability/capacity
+situations, such as raffles or sign-ups to events. Related to that, we expanded the statistics to allow exporting
+the successfully registered submissions.
+
+**🤖 Automatic technical configuration**
+
+We're shipping some tooling for infrastructure teams that deploy Open Forms - this makes it possible to
+provision some configuration aspects that previously had to be done in the admin interface via point-and-click.
+
+We're still expanding on the covered configuration aspects, so stay tuned for more!
+
+**🚸 User Experience improvements**
+
+We have made a ton of user experience improvements in registration and prefill plugin configurations! No
+more copying of URLs from other systems - instead, you select the relevant option in a dropdown.
+Dropdowns that support a search field to wade through those tens or hundreds of available case types!
+
+And, wherever you need to choose a form variable, you now have the options grouped by type of variable
+*and* the context of where this variable occurs, topped of with a search field.
+
+Detailed changes
+----------------
+
+**New features**
+
+* [#4969] Improved the UX of the form designer:
+
+    - The base form configuration tab now groups related fields and collapses them to declutter the UI.
+    - Moved the introduction page configuration to clarify the difference with the introduction text fields.
+* Registration plugins:
+
+    * [:backend:`4686`] All the registration plugin configuration options are now consistently managed in a 
+      modal with better UX.
+
+    * Email:
+
+        * [:backend:`4650`] The email registration plugin now allows setting the recipient using form variables.
+    * Objects API:
+
+        * [:backend:`4978`] Made v2 config the default one.
+        * Updated technical configuration documentation for Objects API.
+        * [:backend:`4398`] The object ownership is now checked, when a user is authenticated, during prefill
+          and pre-registration.
```

Comment:
```suggestion
        * [:backend:`4398`] You can now update a referenced existing object rather than create a new record.
          When the object is being updated, the BSN of the authenticated user is verified against the existing
          object data.
```